src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+// Constants:
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+// initializes state
+const initialState = [];
+
+// action creators
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
```

Comment:
### Optional
- [ ] It would be better to pass the `bookId` to the removeBook function since we are just removing a book and not returning any data

Reply:
@addod19 Okay I'll have that sorted. Thank you 💯